src/AltinnCore/Designer/Controllers/RepositoryController.cs

```diff
@@ -311,35 +311,20 @@ public ActionResult<HttpResponseMessage> StageChange(string org, string reposito
         /// </summary>
         /// <param name="org">Unique identifier of the organisation responsible for the app.</param>
         /// <param name="repository">The name of repository.</param>
-        /// <param name="appTitle">The title of the app to create.</param>
         /// <returns>
         /// An indication if app was created successful or not.
         /// </returns>
         [Authorize]
         [HttpPost]
-        public RepositoryModel CreateService(string org, string repository, string appTitle)
+        public RepositoryModel CreateApp(string org, string repository)
         {
-            ServiceConfiguration serviceConfiguration = new ServiceConfiguration
+            var config = new ServiceConfiguration
```

Comment:
We usually try to avoid using var.. could we rather use `ServiceConfiguration config =` here?

Reply:
Avoiding var makes sense when the type of a variable is not clear when reading code. For plain variable creation, as in this case, the code readability is increased.